cognite/neat/graph/extractors/_mock_graph_generator.py

```diff
@@ -17,6 +17,7 @@
 from cognite.neat.rules.models.data_types import DataType
 from cognite.neat.rules.models.entities import ClassEntity, EntityTypes
 from cognite.neat.rules.models.information import InformationProperty
+from cognite.neat.rules.transformers import DMSToInformation
```

Comment:
I am always a bit puzzled what to do with all caps acronyms when defining a class.